### PR TITLE
Pass manual static viz colors to frontend

### PR DESCRIPTION
**Pull request by metamben**
This is milestone 5 of #22816.

### CODE REVIEW

File: src/metabase/api/pulse.clj [hunk 1]
```diff
@@ -178,7 +178,10 @@
   [{query :dataset_query, card-id :id}]
   (binding [qp.perms/*card-id* card-id]
     (qp/process-query-and-save-execution!
-     (assoc query :async? false)
+     (-> query
+         (assoc :async? false)
+         (assoc :middleware {:process-viz-settings? true
+                             :js-int-to-string?     false}))
```
Comment (metamben) [inline src/metabase/api/pulse.clj:184]: I suppose the preview should also take into account the viz settings stored in the DB.

File: src/metabase/api/pulse.clj [hunk 2]
```diff
@@ -178,7 +178,10 @@
   [{query :dataset_query, card-id :id}]
   (binding [qp.perms/*card-id* card-id]
     (qp/process-query-and-save-execution!
-     (assoc query :async? false)
+     (assoc query
+            :async? false
+            :middleware {:process-viz-settings? true
+                         :js-int-to-string?     false})
```
Comment (metamben) [inline src/metabase/api/pulse.clj:184]: I suppose the preview should also take into account the viz settings stored in the DB.

**Review by howonlee — APPROVED** [review]

**Review by ranquild — COMMENTED** [review]
@metamben Have you managed to get it working end-to-end? I've tried sending to email and Slack and both don't seem to work.

**Comment by metamben** [comment]
> @metamben Have you managed to get it working end-to-end? I've tried sending to email and Slack and both don't seem to work.

Yes, I tested via email and I got the chart with the custom color.

**Pull request merged**